### PR TITLE
added DrCAO and also AIWeb3 

### PR DESCRIPTION
**Pull request by ahjxcrz**

### CODE REVIEW

**Comment by ahjxcrz** [comment]
update AIWeb3 and DrCAO

**Comment by ahjxcrz** [comment]
https://kusama.subscan.io/extrinsic/17229858-4

File: registry/kusama.json [hunk 1]
```diff
@@ -123,7 +123,7 @@
         "address": "JA9TjkzZzsJnBC8igbnLZ2WYvgsYHd6fu54QJAViqnADoZq",
         "name": "DOZENODES",
         "image": "https://raw.githubusercontent.com/nova-wallet/opengov-delegate-registry/master/images/dozenodes.png",
-        "shortDescription": "Kusama Validator Operator www.dozenodes.com",
+        "shortDescription": "Kusama Validator Operator www.dozendoes.com",
```
Comment (spazcoin) [inline registry/kusama.json:126]: this change looks like an accident.

**Comment by stepanLav** [comment]
Thank you for your request!
@ahjxcrz    

You can check how it will look like:

<details>
  <summary>App screens</summary>

Screenshot 2023-03-28 at 11 14 08
Screenshot 2023-03-28 at 11 14 12
Screenshot 2023-03-28 at 11 14 33
Screenshot 2023-03-28 at 11 14 39
Screenshot 2023-03-28 at 11 14 42


</details>

As you want to add two accounts, could you also submit an extrinsic for `HPD6vbDL87yEFkKcpzLKuTbkQrvQXWMwbEzegvKvYDHVcT2`?
After that, if you are happy with how your accounts look like, we will approve your request.

**Comment by ahjxcrz** [comment]
sure, for the AIWeb3, the twitter can be updated to @aiweb3dao

**Comment by ahjxcrz** [comment]
https://kusama.subscan.io/extrinsic/17237893-2

File: registry/kusama.json (hunk 1 above)
Comment (stepanLav) [inline registry/kusama.json:126]:
```suggestion
        "shortDescription": "Kusama Validator Operator www.dozenodes.com",
```

**Review by stepanLav — APPROVED** [review]

**Pull request merged**